lib/morse_translator.rb

```diff
@@ -40,4 +40,24 @@ def initialize
                     " " => " "}
   end
 
+  def translate(text)
+    char_arr = text.chars
+    encoded = char_arr.map do |char|
+      @dictionary[char.downcase]
+    end
+
+    encoded.join
+  end
+
+  def morse_to_eng(text)
+    reverse_dictionary = @dictionary.invert
+
+    char_arr = text.split(" ")
+    decoded = char_arr.map do |char|
+      reverse_dictionary[char]
+    end
+
+    decoded.join
```

Comment:
Close! This doesn't take into account the double spaces in the morse input being translated into single spaces in the output.